jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlField.java

```diff
@@ -122,6 +125,15 @@ static void setFieldOrThrow(Field field, Object newInstance, Object value) {
             }
         }
 
+        if (clazz.equals(BigDecimal.class)) {
+            DecimalFormat decimalFormat = getDecimalFormat();
+            try {
+                return (U) decimalFormat.parse(value);
+            } catch ( ParseException e ) {
```

Comment:
https://github.com/DroidsOnRoids/jspoon/pull/20#discussion_r159078391

Reply:
Ha, my current IDE settings is generous of spaces. 😅 

Which reminds me, a checkstyle config would be nice!